docs/UserGuide.adoc

```diff
@@ -247,11 +247,16 @@ Format: `/order find [n/NAME] [p/PHONE] [a/ADDRESS] [f/FOOD] [dt/DATETIME] [st/O
 * All fields are optional. However, at least one field must be specified.
 * Empty value after any fields are not allowed. e.g. `n/`
 * The search is case insensitive. e.g `tom` will match `Tom`
-* Partial match is allowed for name, phone and food. e.g. `alex` will match `alex lim` and `al` will match `alex lim`
-* To find for orders from a specific date, use 1 date field. e.g `dt/01-10-2018 10:00:00` will return order/s on that
+* Partial match is allowed for name, phone, food and address. e.g. `alex` will match `alex lim` and `al` will match `alex lim`
+* When more than 1 fields are given, the find will do an `and` on all fields. e.g. `n/alex p/1234`, will be `n == 'alex' AND p == 1234`
```

Comment:
Maybe can state here that date is an exception? 

Like e.g. except for date field. Refer below for how that works

Reply:
It stated on line 252 to 259. Or is it still unclear?

Reply:
No its okay! I thought you could maybe signpost here as well (so people who sometimes dont read the whole thing are aware)